files/build_templates/qos_config.j2

```diff
@@ -106,6 +115,9 @@
         }
     },
 {% endif %}
+{% if (generate_pfc_to_queue_map is defined) %}
```

Comment:
Any case we need to define generate_pfc_to_queue_map rather than use the default 1-1 PFC to queue mapping?

Reply:
No specific override is defined. A placeholder for future.